src/main/java/org/fundacion/jala/converter/models/facade/ChecksumFacade.java

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ *
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ *
+ * @author Gustavo Zacarias Huanca Alconz
+ */
+package org.fundacion.jala.converter.models.facade;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.fundacion.jala.converter.models.Asset;
+import org.fundacion.jala.converter.service.FileStorageService;
+import org.springframework.web.multipart.MultipartFile;
+import java.io.IOException;
+import java.security.NoSuchAlgorithmException;
+import java.util.List;
+import java.util.stream.Collectors;
+import static org.fundacion.jala.converter.models.AssetSQL.listAsset;
+import static org.fundacion.jala.converter.service.ChecksumService.getFileChecksum;
+
+public class ChecksumFacade {
+    private static final Logger LOGGER = LogManager.getLogger();
+
+    private ChecksumFacade() {
```

Comment:
You should remove this empty constructor.

Reply:
No, this constructor is necessary not to do instances the class